packages/nuxt-locokit/playground/pages/404.vue

```diff
@@ -1,6 +1,40 @@
 <template>
-  <div>
-    404 Page
-    <p>{{ $route.params.slug }}</p>
+  <div
+    class="h-screen bg-cover bg-no-repeat bg-center"
+    :style="{
+      background:
+        'bottom center / contain no-repeat url' +
+        '(' +
+        runtimeConfig.public.ERROR_BACKGROUND_IMAGE_URL +
+        ')' +
+        'var(--color-primary)',
+    }"
+  >
+    <div class="container mx-auto text-center justify-center flex-1">
+      <span class="text-white pt-8 mb-2 text-3xl text-9xl font-black sr-only"
+        >404</span
```

Comment:
We need to display a real sentence so that everyone can understand it. You can use the `title` here.

Reply:
done